releases/v16.rst

```diff
@@ -0,0 +1,28 @@
+#######################
+LSST Stack 16.0 Release
+#######################
+
+Version 16.0 of the LSST Science Pipelines is now available. It is based on 2018 weekly 10.
+
+LSST Pipelines
+==============
+
+Full details of the LSST Science Pipelines 15.0 release can be found
+in the :ref:`extensive release notes <release-v16-0>`.  We have also
+published a `Characterization Metric Report <https://ls.st/DMTR-tbd>`_.
```

Comment:
You'll want to hold this PR until the characterization report is published so you make sure this URL is correct.

Reply:
We can easily issue the docushare handle at any time.

Reply:
I was considering to use tbd, in order to make clear that need to be updated before merging the 16.0 branch into the trunk. 
Simon told me that the characterization report should be ready this week.
This information should not be required now.